zeppelin-web/src/app/notebook/notebook.css

```diff
@@ -221,3 +232,41 @@
   text-decoration: none;
   cursor: default;
 }
+
+.userlist {
+  width: 320px;
```

Comment:
Yes, it is too long compared to the box. I have reduced the size in commit [b7333d6](https://github.com/apache/incubator-zeppelin/pull/937/commits/b7333d66870d9f106640583904a492212ec631b4)